docs/config.md

```diff
@@ -0,0 +1,89 @@
+# Configuration
+
+These are the config settings available to jj/Jujutsu.
+
+The config settings are located at `~/.jjconfig.toml`.
+
+See the [TOML site](https://toml.io/en/) for more on syntax,
+but one thing to remember is that anything under a heading can be dotted
+e.g. `user.name = "YOUR NAME"` is equivalent to:
+
+    [user]
+      name = "YOUR NAME"
+
+Headings only need to be set once in the real config file but the examples below repeat the headings so it's clear where they go.
+
+The other thing to remember is that the value of a setting (the part to the right of the `=` sign) should be surrounded in quotes if it's a string. That's probably enough TOML to keep you out of trouble but the syntax guide is very short if you ever need to check.
+
+
+## User settings
+
+    [user]
+      name = "YOUR NAME" 
+      email = "YOUR_EMAIL@example.com"
+
+Don't forget to change these to your own details!
+
+
+## UI settings
+
+These all go under the `[ui]` heading.
+
+
+### Colorizing output
+
+Possible values are `always`, `never` and `auto` (default: `auto`). 
+`auto` will use color only when writing to a terminal. 
+
+This setting overrides the `NO_COLOR` environment variable (if set).
+
+    [ui]
+      color = "never" # Turn off color
+
+
+### Editor
+
+The default editor is set via `ui.editor`,
+though there are several places to set it. The priority is as follows:
+`$JJ_EDITOR` env var > `ui.editor` > `$VISUAL` env var > `$EDITOR` env var
```

Comment:
Did you mean to drop the "env var" (since the `$` probably suggest that strongly enough)?

Reply:
I left it in as I'm keeping newer users (to shell/VCS) in mind. I'll have a go at restructuring it so I can mention it without repeating it.

Reply:
Done this, let me know what you think.

Reply:
LGTM